tests/queries/0_stateless/02016_aggregation_spark_bar.reference

```diff
@@ -1,26 +1,50 @@
-▁
+-- { echoOn }
+
+SELECT sparkbar(2)(event_date,cnt) FROM spark_bar_test;
 ▁█
+SELECT sparkbar(3)(event_date,cnt) FROM spark_bar_test;
 ▃█▁
+SELECT sparkbar(4)(event_date,cnt) FROM spark_bar_test;
 ▄▅█▁
+SELECT sparkbar(5)(event_date,cnt) FROM spark_bar_test;
 ▄▄█▇▁
+SELECT sparkbar(6)(event_date,cnt) FROM spark_bar_test;
 ▃▄▅█▃▁
+SELECT sparkbar(7)(event_date,cnt) FROM spark_bar_test;
 ▂▅▃▇█▁▂
+SELECT sparkbar(8)(event_date,cnt) FROM spark_bar_test;
 ▂▅▃▅██ ▁
+SELECT sparkbar(9)(event_date,cnt) FROM spark_bar_test;
 ▁▅▄▃██▅ ▁
+SELECT sparkbar(10)(event_date,cnt) FROM spark_bar_test;
 ▁▄▄▂▅▇█▂ ▂
```

Comment:
This histogram doesn't look correct:
in the data there we have ...(7,'2020-01-06'),(6,'2020-01-07'),(8,'2020-01-08'),(2,'2020-01-11') so there should not be something like 2 in bucket 7.
Looks like test results need to be updated

Reply:
Bucket 7 corresponds to `2020-01-06` or `2020-01-07` that have weight 6 or, the maximal

Reply:
Difference between old and new implementation [`c6e473a` (#45961)](https://github.com/ClickHouse/ClickHouse/pull/45961/commits/c6e473a66ae581fe0622aa50716ac36a9758b72a)